### PR TITLE
tried to fix bug with logo

### PR DESCRIPTION
**Pull request by katsmamina**
the logo was not displayed on GitHub pages, while it was very well displayed locally. I checked that it can be connected to the register of the file-name and changed it to lowercase. not sure it will work, but we will see it after merging :D

### CODE REVIEW

**Review by danielhalasz — CHANGES_REQUESTED** [review]
in the index.html, the class name has to be changed on line 18
          ```logoQuiz```

also, the png is not there in the assets folder, so that could be the problem :)

**Comment by katsmamina** [comment]
thank you @danielhalasz ! indeed, I forgot to save html file, fixed that.
as for the png file in the assets folder, it's there, I see it, but as mentioned before, the problem is its name (LogoJSQ). I tried to edit the name, but apparently it didn't work, so I just uploaded the same file with an amended name. hope it will work .

**Comment by danielhalasz** [comment]
@katsmamina the png file is not in the assets folder, no matter how hard I look :) 
in the 3-select-interaction branch the logo is there, but not in this branch's assets folder

**Comment by WalterAlvar** [comment]
I think the problem it is at line 18 of `index.html` file, its a very silly one 😆 we just need to add a period ( . ) at the begin of the "src:"
`src="/client/assets/LogoJSQ.png"` and just make it:  `src="./client/assets/LogoJSQ.png"` this is the way the favicon it's display, and also how do we call the css files in the head.

And yes, the files are there:
https://github.com/katsmamina/architecture-quiz-group2/tree/master/client/assets

File: index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
       <!-- This is the main page -->
       <div id="quiz-header" class="quiz-header">
         <div id="logo" class="logo">
-          <img class="logo-JSQ" src="/client/assets/LogoJSQ.png" alt="logoQuiz">
+          <img class="logo-jsq" src="/client/assets/Logo-jsq.png" alt="logoQuiz">
```
Comment (WalterAlvar) [inline index.html]: just add a period in `src="./client/assets/Logo-jsq.png" ` before the first slash ' / '

**Review by WalterAlvar — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (danielhalasz) [reply index.html]: oh, okay, true, the file was there on the master branch, just not in the bug-fix :) after updating master, it works for me, even without the dot :)

**Review by danielhalasz — APPROVED** [review]

**Pull request merged**